client/src/Palette.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+function Palette(props) {
+  const [paletteItems, setPaletteItems] = React.useState();
+
+  React.useEffect(() => {
+    fetch(`/profiles`)
+      .then((res) => res.json())
+      .then((data) => {
+        setPaletteItems(data.palettes);
+      });
+  }, []);
+
+  return (
+    <div className="flex">
+      {paletteItems &&
+        paletteItems.map((palettes) => (
+          // TODO: - Add key with ID after having real DB connection
+          <div className="relative m-3">
```

Comment:
Sorry I think my last comment was confusing. We do want some sort of key here, but in the future, an ID will be better than the index

```suggestion
          <div className="relative m-3  key={index}">
```